netlink.go

```diff
@@ -351,6 +351,17 @@ func assembleService(attrs []syscall.NetlinkRouteAttr) (*Service, error) {
 
 	}
 
+	// in older kernels (< 3.18), the svc address family attribute may not exist so we must
+	// assume it based on the svc address provided.
+	if s.AddressFamily == 0 {
+		addr := (net.IP)(addressBytes)
+		if addr.To4() != nil {
```

Comment:
In running a few more tests, I realized this won't work because `To4()` checks the last 4 bytes for v4 addresses and leading 0 but netlink returns IPv4 addresses in the first 4 bytes. 

Will follow-up with a fix.